scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -56,8 +56,16 @@ static u32 preempt_cursor;
 
 extern unsigned CONFIG_HZ __kconfig;
 
-#define dbg(fmt, args...)	do { if (debug) bpf_printk(fmt, ##args); } while (0)
-#define trace(fmt, args...)	do { if (debug > 1) bpf_printk(fmt, ##args); } while (0)
+#define dbg(fmt, args...)            \
```

Comment:
No bueno on this

Reply:
really? thought that was one of the nicer bits, alright will see about fixing that.